go.mod

```diff
@@ -1,11 +1,14 @@
 module github.com/openshift/prom-label-proxy
 
-go 1.12
+go 1.13
+
+// v2.18.1
+replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.0.0-20200507164740-ecee9c8abfd1
```

Comment:
I'd rather have `github.com/prometheus/prometheus v1.8.2-0.20200507164740-ecee9c8abfd1` in the `require` section and avoid using `replace`. WDYT?

Reply:
Thanks for the pointer!